sql_generators/mobile_kpi_support_metrics/templates/active_users.view.sql

```diff
@@ -3,7 +3,10 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ dataset }}.{{ name }}`
 AS
 SELECT
-  * EXCEPT (isp),
+  * EXCEPT (isp) REPLACE(
+    -- Lower device_manufacturer as in some cases the same manufacturer value has different casing.
+    LOWER(device_manufacturer) AS device_manufacturer
```

Comment:
Lowering device_manufacturer here so that everything downstream inherits the lowered values.